src/lib/storage/refinements.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+#
+# encoding: utf-8
+
+# Copyright (c) [2016] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+#
+# add some useful methods to libstorage-ng objects
+#
+
+require "storage"
+
+module Storage
+ class Disk
+      def inspect
```

Comment:
For things that will eventually appear in libstorage in a future (like #inspect or #partition_table?), I agree that monkey patching is the way to go. No reason to create a refinement that you need to explicitly activate everywhere. Just patch the class for the time being.